src/contexts/auth/types.ts

```diff
@@ -1,5 +1,13 @@
 import { AuthState } from './AuthContext';
 
+export const LOGGED_OUT = Symbol('LOGGED_OUT');
+export const LOADING = Symbol('LOADING');
+export const UNKNOWN = Symbol('UNKNOWN');
+
 export function isLoggedInState(state: AuthState) {
   return typeof state === 'object';
 }
```

Comment:
we should be a bit more explicit here, since `null` in javascript technically is a typeof object. let's use this:

```js
return typeof state === 'object' && state !== null && 'jwt' in state
```

that last part comes from the fact that `type LoggedIn = { jwt: string };`, so we not only wanna check for object but the fact that `jwt` key exists in the object

we also might as well move `type LoggedIn` into this file

Reply:
done ✅